.github/workflows/update-parser-tests.yml

```diff
@@ -57,27 +57,35 @@ jobs:
           github-token: ${{ secrets.BOT_TOKEN }}
           script: |
             const base = process.env.GITHUB_REF.replace("refs/heads/", "");
-            const { data: prs } = await github.pulls.list({
+            const requestParam = {
               owner: context.repo.owner,
               repo: context.repo.repo,
-              head: "update-test262-parser",
-              base: base
-            });
+              head: context.repo.owner + ":update-test262-parser",
```

Comment:
Here is the critical fix: It accepts `user:branch` only: https://octokit.github.io/rest.js/v18#pulls-list

When `branch` is passed, it returns all the open PRs. 🤷‍♂️

Reply:
I wish octokit would validate them :shrug: